static/js/sponsors/applicationForm.js

```diff
@@ -47,6 +47,8 @@ $(document).ready(function(){
           let packageOnlyBenefit = $(this).attr("package_only");
           if (packageOnlyBenefit) $(this).attr("disabled", true);
           return;
+      } else {
+          $('label[benefit_id=' + benefit + ']').addClass("active");
```

Comment:
Something about this change seems to have broken the clear selections functionality @berinhard